src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    protected String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Overloaded expression, please introduce some variables. Also Random should be class level variable